src/components/Book/Book.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import classes from './Book.module.css';
+
+const Book = (props) => {
+  const {
+    id, category, title, removeBook,
+  } = props;
+
+  return (
+    <>
+      <tr>
+        <td className={classes.book_card}>
+          <div>
+            <span>{category}</span>
+            <h1>{title}</h1>
+            <ul className={classes.actions_list}>
+              <li>
+                <button type="button">Comments</button>
+              </li>
+              <li>
+                <button type="button" onClick={() => removeBook(id)}>Remove</button>
+              </li>
+              <li>
+                <button type="button">Edit</button>
+              </li>
+            </ul>
+          </div>
+          <div className={classes.completed}>
+            <div>0%</div>
+            <span>Completed</span>
+          </div>
+          <div className={classes.progress}>
+            <p>CURRENT CHAPTER</p>
+            <p>Introduction</p>
+            <button type="button">UPDATE PROGRESS</button>
+          </div>
+        </td>
+      </tr>
+    </>
```

Comment:
- [ ] Since you're expected to produce a similar page, I strongly suggest you improve the `Book` component styles.
  - The category name for each book should be bold and greyed out.
  - The book author should be included. This can be the same static text for all books.
  - The percentage text (e.g. 64% Completed) should be beside the loading/status circle, not inside the circle.
  - The `CURRENT CHAPTER` text should be bold and greyed out.
  - I think you should also work on the spacings between the elements of the component. Please check the image below for a detailed representation of how your book component should be styled.
![book](https://user-images.githubusercontent.com/26856491/119777411-057bdf80-bebe-11eb-98d8-32ab4371041f.png)